### PR TITLE
feat: deprecate uikitOption.groupChannel.enableReactionsSupergroup

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/CLNP-3581

- A deprecation warning from `uikit-tools`
Screenshot 2024-07-01 at 4 47 53 PM

- from `lib/types.ts`
Screenshot 2024-07-01 at 4 36 31 PM

### CODE REVIEW

File: src/lib/types.ts [hunk 1]
```diff
@@ -100,6 +100,9 @@ export interface SendBirdStateConfig {
     enableOgtag: SBUConfig['groupChannel']['channel']['enableOgtag'];
     enableTypingIndicator: SBUConfig['groupChannel']['channel']['enableTypingIndicator'];
     enableReactions: SBUConfig['groupChannel']['channel']['enableReactions'];
+    /**
+     * @deprecated Currently, this feature is turned off by default. If you wish to use this feature, contact us: {@link https://dashboard.sendbird.com/settings/contact_us?category=feedback_and_feature_requests&product=UIKit}
+     */
     enableReactionsSupergroup: SBUConfig['groupChannel']['channel']['enableReactionsSupergroup'];
```
Comment (bang9) [inline src/lib/types.ts:106]:
```suggestion
    enableReactionsSupergroup: never
```

**Comment by bang9** [comment]
+ released in ~-> https://github.com/sendbird/sendbird-uikit-core-ts/releases/tag/v0.0.1-alpha.76

**Review by chrisallo — APPROVED** [review]

File: src/lib/Sendbird.tsx [hunk 2]
```diff
@@ -374,6 +373,7 @@ const SendbirdSDK = ({
             showSuggestedRepliesFor: configs.groupChannel.channel.showSuggestedRepliesFor,
             suggestedRepliesDirection: configs.groupChannel.channel.suggestedRepliesDirection,
             enableMarkdownForUserMessage: configs.groupChannel.channel.enableMarkdownForUserMessage,
+            enableReactionsSupergroup: undefined as never,
```
Comment (bang9) [inline src/lib/Sendbird.tsx:376]: If you modify this here, won't you be unable to receive changes from the remote as well?

Comment (AhyoungRyu) [reply src/lib/Sendbird.tsx:376]: Oh I refer to this change made in RN https://github.com/sendbird/sendbird-uikit-react-native/pull/183/files#diff-c566ff50cbbbf3f800ea001b2af36833f1b5e8bcb484b1ccd2a8237345beeee1R215 saw it's set as undefined. Do you think we need to keep as is ? 
```
enableReactionsSupergroup: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableReactionsSupergroup as never,
```

Comment (bang9) [reply src/lib/Sendbird.tsx:376]: That code is the value injected into UIKitConfigContextProvider, and the code you modified maps the value retrieved from UIKitConfigContext to SendbirdProvider

Comment (AhyoungRyu) [inline src/lib/Sendbird.tsx:376]:
```suggestion
            enableReactionsSupergroup: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableReactionsSupergroup as never,
```

Comment (AhyoungRyu) [reply src/lib/Sendbird.tsx]: Oh I got confused. Just updated :)

**Review by bang9 — APPROVED** [review]
lgtm!

**Pull request merged**